src/app.rs

```diff
@@ -103,7 +103,8 @@ pub fn app() -> App<'static, 'static> {
         .arg(flag("unrestricted").short("u")
              .multiple(true))
         .arg(flag("invert-match").short("v"))
-        .arg(flag("word-regexp").short("w"))
+        .arg(flag("word-regexp").short("w").overrides_with("line-regexp"))
```

Comment:
It should actually be fine to apply both the `word-regexp` and `line-regexp` pattern transformations at once, since `\b` also matches the beginning/end of line. But i figured it'd be cleaner and more future-proof to make the one override the other.